packages/example/src/components/Notifications/History.tsx

```diff
@@ -0,0 +1,201 @@
+import type { Notification } from '@usedapp/core'
+import { ChainId, useNotificationsContext, useTransactionsContext } from '@usedapp/core'
+import type { ReactNode } from 'react'
+import styled from 'styled-components'
+import { TextBold } from '../../typography/Text'
+import { ContentBlock } from '../base/base'
+
+interface TableWrapperProps {
+  children: ReactNode
+  title: string
+}
+
+const TableWrapper = ({ children, title }: TableWrapperProps) => (
+  <ContentBlock>
+    <CellTitle>{title}</CellTitle>
+    <Table>{children}</Table>
+  </ContentBlock>
+)
+
+interface TransactionListProps {
+  chainId: ChainId
+}
+
+export const TransactionsList = ({ chainId }: TransactionListProps) => {
+  const { transactions } = useTransactionsContext()
+  const chainTransactions = transactions[chainId] ?? []
+  const reversed = [...chainTransactions].reverse()
```

Comment:
Let's do reverse by default (modify reducer).

Idea to discuss:
```ts
useTransactions({chainId: ...})
```
**ChainId**
- chainId is by default null, in which case useTransaction will use useEthers to determine chainId.
- chainId can be single chainId 
- in the future: array of chainIds
- question: how go we show all chainIds? empty array? 0? other?



```ts
{chainId: null }
```